modules/external-auth-disabling.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * authentication/external-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-auth-disabling_{context}"]
+= Disabling direct authentication
+
+If necessary, you can disable direct authentication for your cluster and revert back to authenticating with the built-in OpenShift OAuth server.
+
+.Prerequisites
+
+* You have access to the `kubeconfig` file for the cluster.
+
+.Procedure
+
+. Log in to the cluster by using your `kubeconfig` file.
++
+You must use the `kubeconfig` file to configure the direct authentication. If you use a token, the token can become invalid during the process of the procedure.
+
+. Update the authentication configuration to use the built-in OpenShift OAuth server by running the following command:
++
+[source,terminal]
+----
+$ oc patch authentication.config/cluster --type=merge -p='
+spec:
+  type: "" <1>
+  oidcProviders: null <2>
+'
+----
+<1> Sets `type` to `""` to use the built-in OpenShift OAuth server.
+<2> Removes the `oidcProviders` configuration.
+
+.Verification
+
+. Wait for the cluster to roll out new revisions to all nodes.
+
+.. Check the Kubernetes API server Operator status by running the following command:
++
+[source,terminal]
+----
+$ oc get co kube-apiserver
+----
++
+.Example output
+[source,terminal]
+----
+NAME             VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+kube-apiserver   4.19.0    True        True          False      85m     NodeInstallerProgressing: 2 node are at revision 12; 1 node is at revision 14
+----
++
+The message in the above example shows that one node has progressed to the new revision and two nodes have not yet updated. It can take 20 minutes or more to roll out the new revision to all nodes, depending on the size of your cluster.
```

Comment:
Same comment as before regarding "above".

Reply:
Updated to "preceding"